kvmagent/kvmagent/test/test_deip.py

```
@@ -519,5 +522,36 @@ def test_delete_v6_cleans_legacy_gw_chain(self):
+            mock.patch("zstacklib.utils.shell.get_process"),
+        ]
+        mock_iproute = patchers[0].start()
+        mock_linux = patchers[1].start()
+        mock_get_process = patchers[2].start()
+        try:
+            mock_iproute.IpNetnsShell.list_netns.return_value = []
+            mock_linux.is_network_device_existing.return_value = False
+            mock_get_process.side_effect = _make_fake_process([])
+
+            eip_cmd = Eip()
+            method = getattr(type(eip_cmd), "delete_eip_with_ns")
```

Ruff (0.15.14)

[warning] Do not call `getattr` with a constant attribute value. It is not any safer than normal property access.

Replace `getattr` with attribute access

(B009)